spec/views/emails/send_email.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe "emails/email_form", type: :view do
+  before(:each) do
+    @email = assign(:email, FactoryGirl.build(:email))
+    sign_in(FactoryGirl.create(:user, role: :admin))
+    render :partial => "emails/email_form"
+  end
+
+  it("renders required email fields") do
+    expect(rendered).to have_field(t'emails.email_form.show_recipients')
```

Comment:
Inside View tests you can use the abbreviated version, but I will update to make it clearer